server/internal/app/file.go

```diff
@@ -46,6 +46,20 @@ func serveFiles(
 		}),
 	)
 
+	ec.GET(
+		"/export/:filename",
+		fileHandler(func(ctx echo.Context) (io.Reader, string, error) {
+			filename := ctx.Param("filename")
+			r, err := repo.ReadExportProjectZip(ctx.Request().Context(), filename)
+			if err != nil {
+				return nil, "", rerror.ErrNotFound
+			}
+			// download and then delete
+			err = repo.RemoveExportProjectZip(ctx.Request().Context(), filename)
+			return r, filename, err
+		}),
+	)
+
```

Comment:
We wouldn't want to move zip file to become downloadable from a URL, its should be one time thing.

Reply:
@pyshx I've set up the export through GraphQL, but GraphQL does not support file downloads.

Currently, I make a request with the project ID, create a ZIP file, temporarily store it in storage, and then return the download path in the response.

Would it be better to use a REST API to handle the request and return the ZIP file directly?

Reply:
@hexaforce I understood, Serving files directly through Echo avoids the overhead of encoding files into Base64 and transmitting them through GraphQL.

Reply:
It's okay to leave it as is.